kyaml/openapi/openapi.go

```diff
@@ -389,7 +393,7 @@ func (rs *ResourceSchema) PatchStrategyAndKey() (string, string) {
 const (
 	// kubernetesAPIDefaultVersion is the latest version number of the statically compiled in
 	// OpenAPI schema for kubernetes built-in types
-	kubernetesAPIDefaultVersion = kubernetesapi.DefaultOpenApi
+	kubernetesAPIDefaultVersion = kubernetesapi.DefaultOpenAPI
```

Comment:
The name should keep consistent with `kubernetesOpenAPIVersion`

Reply:
changed to kubernetesOpenAPIDefaultVersion